va/va.go

```diff
@@ -229,7 +229,7 @@ func NewValidationAuthorityImpl(
 	accountURIPrefixes []string,
 ) (*ValidationAuthorityImpl, error) {
 
-	if features.Enabled(features.CAAAccountURI) && len(accountURIPrefixes) == 0 {
+	if len(accountURIPrefixes) == 0 {
```

Comment:
Should we also add a validation to `Config.VA.AccountURIPrefixes` in `cmd/boulder-va/main.go`?

Reply:
Good idea, done.